ktor-client/ktor-client-features/ktor-client-tracing/jvm/test/TracingWrapperTest.kt

```diff
@@ -0,0 +1,251 @@
+/*
+ * Copyright 2014-2020 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+import io.ktor.client.engine.*
+import io.ktor.client.features.tracing.*
+import io.ktor.client.request.*
+import io.ktor.http.*
+import io.ktor.http.cio.websocket.*
+import io.ktor.http.content.*
+import io.ktor.util.*
+import io.ktor.util.date.*
+import junit.framework.Assert.*
+import kotlinx.coroutines.channels.*
+import kotlinx.coroutines.*
+import org.junit.*
+import kotlin.coroutines.*
+
+class TracingWrapperTest {
+    @Test
+    fun testOutgoingChannelTracerOffer() = runBlocking {
```

Comment:
Could you add integration tests as well?

Reply:
I added a simple integration test.